examples/hello_triangle_c/CMakeLists.txt

```diff
@@ -27,4 +34,4 @@ find_library(WGPU_LIBRARY wgpu_native
     HINTS "${CMAKE_CURRENT_SOURCE_DIR}/../../target/debug"
 )
 
-target_link_libraries(${TARGET_NAME} ${GLFW_LIBRARY} ${WGPU_LIBRARY} ${OS_LIBRARIES})
+target_link_libraries(${TARGET_NAME} ${GLFW_LIBRARY} ${WGPU_LIBRARY} ${OS_LIBRARIES})
```

Comment:
We should have a newline at the end

Reply:
Habits I guess, I always clean the new lines up. Is there a reason for why this standard is chosen?

Reply:
Some tools expect it to be there to function correctly, so it’s generally best practice to have a new line here. See https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file/5813359 for some examples. GitHub also warns about this in the diff.